### DISCUSSION TITLE
Can't send audio message in API Inbox

### DISCUSSION THREAD

**Discussion by jefranchin** [chatwoot/chatwoot#9864] [Self hosted]
First of all, I would like to congrats the team for the excellent work.

As the title says, when i try to send audio messages in an API Inbox i got the message "The message could not be sent. Please check your connection.". 

If i do the same in a Website Inbox, everything works fine.

![screen-2024-07-31-132613](https://github.com/user-attachments/assets/fa09c566-6a00-4029-99a8-33c1952bfb75)

**Comment by sojan-official** [comment]
Are you using any external integrations in your Chatwoot instance ? 
ie webhooks, agent bots etc 

This message is not generated by Chatwoot.
